components/molecule/accordion/src/styles/index.scss

```diff
@@ -107,8 +107,13 @@ $base-class-item-panel: '#{$base-class-item}Panel';
     font-family: $ff-molecule-accordion-header;
     overflow: hidden;
     text-align: inherit;
-    text-overflow: ellipsis;
-    white-space: nowrap;
+    &--wrap {
+      white-space: wrap;
+    }
+    &--noWrap {
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
```

Comment:
🍆 would it make sense to keep one of the styles as the default one? 🤔 we could skip one of the two modifiers this way

Reply:
Personally, I believe that the management of modifiers can be done in both ways and both are correct.
The positioning of the icon is the same approach. In that case we could also use as default the right one and pass a prop isLeftIconPosition and it would do the same. In the case of the icon there is not going to be any other case than left or right and the modifier is always applied to indicate the positioning.

In case we see a big detriment to do it this way we can change it, but I personally do not see any inconvenience in doing it with modifiers that indicate that type of wrap has the label, besides that it gives us the possibility to apply more cases if in the future it is necessary.